### ISSUE TITLE
NotSupportedException should clarify which project failed

### ISSUE THREAD

**Issue by BrannonKing** [dotnet/try-convert#273]
I've been attempting to convert a solution that contains about 400 project files. About half way through it fails with this error below. The difficulty is that it doesn't say which particular project failed. It might be nice to have a log message stating which project it was attempting to convert.
```
System.NotSupportedException: This project has custom imports in a manner that's not supported.
   at MSBuild.Abstractions.MSBuildWorkspace.CreateSdkBaselineProject(String projectFilePath, IProject project, IProjectRootElement root, ImmutableDictionary`2 configurations) in /_/src/MSBuild.Abstractions/MSBuildWorkspace.cs:line 100
   at MSBuild.Abstractions.MSBuildWorkspace..ctor(ImmutableArray`1 paths, Boolean noBackup) in /_/src/MSBuild.Abstractions/MSBuildWorkspace.cs:line 49
   at MSBuild.Abstractions.MSBuildWorkspaceLoader.LoadWorkspace(String path, Boolean noBackup) in /_/src/MSBuild.Abstractions/MSBuildWorkspaceLoader.cs:line 45
   at MSBuild.Conversion.Program.Run(String project, String workspace, String msbuildPath, Boolean diffOnly, Boolean noBackup) in /_/src/try-convert/Program.cs:line 73
```

**Comment by BrannonKing** [comment]
Issue #124 helped me track down which project was failing.

**Comment by cartermp** [comment]
Thanks for filing. I think this would be perfectly fine to add as an enhancement. Probably a fairly easy adjustment.

**Comment by dotnokato** [comment]
I can work on that.
Do we want to include full path or just the project file name?

**Comment by jmarolf** [comment]
I assume that giving the full project path would be the most helpful

**Comment by dotnokato** [comment]
@BrannonKing Can you check on current version from master if this is what you needed?

**Comment by cartermp** [comment]
I'll close this out - thanks @dotnokato for the fix in #274
